pkg/backends/otlp/internal/data/scope_metrics_test.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1metrics "go.opentelemetry.io/proto/otlp/metrics/v1"
+)
+
+func TestNewScopeMetrics(t *testing.T) {
+	t.Parallel()
+
+	empty := NewScopeMetrics(
+		NewInstrumentationScope("gostatsd/aggregation", "test"),
+	)
+	assert.Equal(
+		t,
+		ScopeMetrics{
+			raw: &v1metrics.ScopeMetrics{
+				Scope:   NewInstrumentationScope("gostatsd/aggregation", "test").raw,
+				Metrics: make([]*v1metrics.Metric, 0),
+			},
+		},
+		empty,
+	)
+
+	sm := NewScopeMetrics(
+		NewInstrumentationScope("gostatsd/aggregation", "test"),
+		NewMetric("my-awesome-metric"),
+	)
+	assert.Len(t, sm.raw.Metrics, 1, "Must have one metric set")
```

Comment:
Should we also assert its equality to the metric passed in on [L29](https://github.com/atlassian/gostatsd/pull/653/files#diff-5a74a3cd8cea5f055070e57ff0e99de842608140d97ce9477771fbc623d3964aR29)

Reply:
I didn't have it in me to write out with the raw proto value that the content matched.